sphinx-docs/source/docs/innereye_deeplearning.md

```diff
@@ -0,0 +1,160 @@
+# InnerEye-DeepLearning
```

Comment:
Why isn't this just `README.md`?

Reply:
To avoid confusion with the actual README - the contents of this file is no longer the same as the README in the GitHub!

Reply:
Would it make sense to merge them into a single README?